includes/class-wc-calypso-bridge-addons.php

```diff
@@ -62,7 +76,9 @@ public function addons_menu() {
 	 * Init the addons page.
 	 */
 	public function addons_page() {
+		require_once WC_CALYSPO_BRIDGE_PLUGIN_PATH . '/includes/class-wc-calypso-bridge-addons-screen.php';
```

Comment:
Typo in the constant -- this probably applies in other places too.
```suggestion
		require_once WC_CALYPSO_BRIDGE_PLUGIN_PATH . '/includes/class-wc-calypso-bridge-addons-screen.php';
```